src/commands/secrets/addSecret.ts

```diff
@@ -12,15 +12,17 @@ export function registerAddSecret(context: vscode.ExtensionContext) {
       const {gitHubRepoContext} = args;
 
       const name = await vscode.window.showInputBox({
-        prompt: "Enter name for new secret"
+        prompt: "Enter name for new secret",
+        ignoreFocusOut: true
       });
 
       if (!name) {
         return;
       }
 
       const value = await vscode.window.showInputBox({
-        prompt: "Enter the new secret value"
+        prompt: "Enter the new secret value",
+        ignoreFocusOut: true
```

Comment:
Is there a way I could/should test this with a test or do we test UI stuff like this manually only? I didn't see any tests verifying the text in the repo

Reply:
From pairing call, this is not setup yet so there aren't tests to add.